networking/p2p/src/main/java/tech/pegasys/teku/networking/p2p/libp2p/gossip/GossipHandler.java

```diff
@@ -87,6 +105,7 @@ public GossipHandler(
       return VALIDATION_IGNORED;
     }
     LOG.trace("Received message for topic {}: {} bytes", topic, bytes.size());
+    bytesReceivedCounter.inc(bytes.size());
```

Comment:
We should be incrementing this prior to the `!processedMessages.add` check. Even though we discard the duplicate gossip messages they still used bandwidth.